cmd/uhc/cluster/list/list.go

```diff
@@ -43,6 +45,43 @@ var Cmd = &cobra.Command{
 	Run:   run,
 }
 
+// return "NONE" if given string is empty
```

Comment:
In Go documentation comments should start with the name of the commented thing, for example:

```go
// checkES returns the string `none` if the given string is empty.
```